packages/opentelemetry-types/src/trace/tracestate.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export class TraceState {
```

Comment:
Similar comments here: should the file be `trace_state.ts`, and should this be an interface? (I realize this is TODO, but wanted to comment anyway on this since this is the `types` package)